models/query_history_enriched.sql

```diff
@@ -24,19 +24,19 @@ query_history as (
     from {{ ref('stg_query_history') }}
 
     {% if is_incremental() %}
-        -- Conservatively re-process the last 7 days to account for late arriving rates data
+        -- Conservatively re-process the last 7 days to account for late arriving rates data. Allow an override from project variable
         -- which changes the cost per query
-        where end_time > (select dateadd(day, -7, max(end_time)) from {{ this }})
+        and end_time > (select dateadd(day, -{{ var('dbt_snowflake_monitoring_incremental_days', '7') }}, max(end_time)) from {{ this }})
```

Comment:
don't think this syntax is valid, need the `where`.

same thing below.

Reply:
adjusted!!